server/src/main/java/org/spine3/server/BoundedContext.java

```diff
@@ -361,7 +361,7 @@ public BoundedContext build() {
             checkNotNull(commandBus, "commandDispatcher must be set");
             checkNotNull(eventBus, "eventBus must be set");
 
-            commandBus.setMultitenant(this.multitenant);
+            commandBus.setIsMultitenant(this.multitenant);
```

Comment:
`setMultitenant` sounds as imperative and is OK for this case. The new name is also harder to read and pronounce. Please revert.